src/elements/card.js

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import LeftHeaderButton from 'component-left-header-button';
+import RightHeaderButton from 'component-right-header-button';
+
+const Card = ({
+  children,
+  title,
+  enableLeftHeader,
+  enableRightHeader,
+  leftHeaderProps,
+  rightHeaderProps,
+  noFade,
+}) => {
+  return (
+    <div className={noFade ? '' : 'card'}>
+      <div className="card-title">
+        <LeftHeaderButton
+          style={{
+            visibility: enableLeftHeader ? 'visible' : 'hidden',
+          }}
+          {...leftHeaderProps}
+        />
+        <div>{title}</div>
+        <RightHeaderButton
+          style={{
+            visibility: enableRightHeader ? 'visible' : 'hidden',
+          }}
+          {...rightHeaderProps}
+        />
+      </div>
+      <div className="card-body">{children}</div>
+    </div>
+  );
+};
+
+Card.defaultProps = {
+  title: 'title',
+  enableLeftHeader: true,
+  enableRightHeader: true,
+  noFade: false,
+  leftHeaderProps: {},
+  rightHeaderProps: {},
+};
+
+export default Card;

```

Comment:
We badly needed this.  There was a lot of copy-pasting cards and header buttons.  I didn't change all of them, but if you touch a file that can use this instead of manually creating the card, change it :)